pkgs/by-name/be/beyond-all-reason/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  fetchurl,
+  appimageTools,
+  openal,
+}:
+let
+  name = "beyond-all-reason-${version}";
+  version = "1.2988.0";
+  pname = "beyond-all-reason";
+
+  src = fetchurl {
+    url = "https://github.com/beyond-all-reason/BYAR-Chobby/releases/download/v${version}/Beyond-All-Reason-${version}.AppImage";
+    hash = "sha256-ZJW5BdxxqyrM2TJTO0SBp4BXt3ILyi77EZx73X8hqJE=";
+  };
+  appimageContents = appimageTools.extractType1 { inherit name src; };
+in
+appimageTools.wrapType2 rec {
```

this rec is unnecessary
```suggestion
appimageTools.wrapType2 {
```